### ISSUE TITLE
Secondary goals not showing up for Medical Chemist

### ISSUE THREAD

**Issue by wanderso** [ParadiseSS13/Paradise#25283]
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1636

### Issue Description

Several shifts in a row I've requested a secondary goal as Medical Chemistry (Pharmacist), using the Request Secondary Goal feature on the newscaster thing. It's gone through - I've been issued a goal crate through cargo, and it says I can't order another secondary goal until I've finished my first one - but I'm never actually told what the goal is. 

### What did you expect to happen?

I expect, when CentComm goes to the trouble of sending me a goal crate for cargo to ignore, to know what's going to go in it.

### What happened instead?

No goal was visible to me.

### Why is this bad/What are the consequences?

Medical chemistry is unable to fulfill secondary goals on their own.

### Steps to reproduce the issue.

1) Be a medical chemist.
2) Request a secondary goal from the console in your office.
3) Try to request another secondary goal, and be told that you can't until you've completed the first one.
4) If Cargo is paying attention, they may ship you your goal crate. Oh, and by the way, the goals back when I could read them said that it needed to be sent to CentComm in a 'locked personal crate', and there are personal crates and goal crates, both of which can be ordered from cargo - you should clarify this.

### When did the problem start happening?

A couple days ago.

### Extra information

No idea what else to say.

### Relevant log output/runtime error

```DM
I got nothing.
```

**Comment by FunnyMan3595** [comment]
Likely fixed by #25170. Please open a new issue if you see this again.
